components/auth0.key.manager/src/main/java/org/wso2/auth0/client/model/Auth0ClientInfo.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.auth0.client.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Auth0ClientInfo {
+    @SerializedName("client_id")
+    private String clientId;
+    @SerializedName("client_secret")
+    private String clientSecret;
```

Comment:
@tharindu1st here too?

Reply:
this was not stored inside therefore no need